src/emqtt_ws.erl

```diff
@@ -45,36 +45,48 @@ connect(Host0, Port, Opts, Timeout) ->
                                         retry => 0}),
     case gun:open(Host1, Port, ConnOpts) of
         {ok, ConnPid} ->
+            MRef = monitor(process, ConnPid),
             case gun:await_up(ConnPid, Timeout) of
```

Comment:
It already sets up a monitor internally, perhaps you've meant?
```suggestion
            case gun:await_up(ConnPid, Timeout, MRef) of
```

Reply:
This is no longer needed now that we handle `gun_down`.  :see_no_evil: